docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -254,22 +257,24 @@ So it is important to check that the code in the release branch is working befor
 Release
 -------
 
-.. warning:: The following steps should be carried out on a :doc:`Flocker development machine <vagrant>`.
-             Log into the machine using SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
+#. Create and log in to a new :doc:`Flocker development machine <vagrant>` using SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
```

Comment:
The next lines tell you to `vagrant up` from the cloned Flocker repository created earlier. Is this not enough?